### ISSUE TITLE
Can the breadcrumb data in the routes be an id?

### ISSUE THREAD

**Issue by MeindertRomkes** [bovandersteene/ngx-breadcrumb#5]
Hello,

In my Routes Module I have an route which I want to give data like

`data: {
    breadcrumb: { :id:}
}` 

Do you guys have any solution for me? Thanks in advance!

**Comment by suppadeliux** [comment]
Yeah, it can be of course an id! but i think the right way to call is just empty, works for me, dont know if thats the correct way:
`{
    path: ':id',
    component: ProjectComponent,
    data: {
      breadcrumb: '',
    },`
But...i actually have a problem, breadcrumb works fine with id, and other properties, the problem starts when i reload the page where the id needs to be shown, the id from the breadcrumb disappear.
Dont know how to solve this problem. Let me know if you have the same problem.
